exercises/concept/lasagna/.docs/instructions.md

```diff
@@ -42,3 +42,4 @@ lasagna = Lasagna.new
 lasagna.total_time_in_minutes(number_of_layers: 3, actual_minutes_in_oven: 20)
 # => 26
 ```
+Dont forget to delete or comment out the instructions in the method before you run your tests to avoid argument errors!
```

Comment:
My suggestion would be to remove this line, and put the message in the exception message as it is raised.  That is, if we decide to keep the exception being raised, as this is a concept exercise and there has been no training at this point for exceptions, from what I can see.